### PR TITLE
Push notification API terraform changes

### PR DESCRIPTION
**Pull request by ssylver93**

### CODE REVIEW

**Review by dhlevi — APPROVED** [review]
Looks consistent with the other changes, will wait for @vivid-cpreston review before merging, as there are a number of other in-flight changes going on

**Review by yzlucas — APPROVED** [review]

**Comment by dhlevi** [comment]
@vivid-cpreston

File: .github/workflows/deploy-dev.yml
Comment (vivid-cpreston) [inline .github/workflows/deploy-dev.yml:131]: Bucket names must be globally unique - we need to create a new bucket, rather than using the one with the same name as WFONE did

File: .github/workflows/deploy-dev.yml
Comment (vivid-cpreston) [inline .github/workflows/deploy-dev.yml:129]: This should refer to a database hosted in the ministry AWS, not the existing Vivid one. There should be an example in wfone-notification-monitors-api

Everything should be using 'dev', not 'int'

File: .github/workflows/deploy-dev.yml
Comment (vivid-cpreston) [inline .github/workflows/deploy-dev.yml:134]: This is an existing, Vivid-owned SQS queue - we'll need to create new queues owned by the ministry AWS account rather than us. There should be examples in wfone-notification-monitors-api

**Review by vivid-cpreston — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-dev.yml
Comment (dhlevi) [reply .github/workflows/deploy-dev.yml:142]: @vivid-cpreston / @ssylver93 This queue should actually already exist from the setup work I did earlier. Take a look at the terraform scripts to verify

**Review by dhlevi — APPROVED** [review]

File: .github/workflows/deploy-dev.yml
Comment (dhlevi) [inline .github/workflows/deploy-dev.yml:154]: We should probably move a bunch of these configs into vars, just so we can change them without PRing changes

Comment (dhlevi) [reply .github/workflows/deploy-dev.yml:154]: and the [INT] one should be removed and replaced with the environment var

File: terraform/rds.tf
Comment (dhlevi) [inline terraform/rds.tf:30]: We'll have to verify the DB configuration once we deploy the notification DB updates. Ths will likely be a schema in the existing DB rather than a new DB?

Comment (ssylver93) [reply terraform/rds.tf:31]: Commented out for now and used the existing datasource config for push notifications, will revisit when db is deployed

**Pull request closed without merging**
